specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_CreateOrUpdate.json

```diff
@@ -13,6 +13,9 @@
         "storageCapacityTiB": 3072,
         "zones" : ["1", "2"],
         "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/scvnet/subnets/sub1",
+        "MSGAddress" : "10.0.0.4",
+        "mountCommand" : "mount -t lustre 10.1.0.13@tcp:/lustrefs /lustre/lustrefs",
+        "lustreVersion" : "2.12.5",
```

Comment:
Should we also have a "lustre client version" field somewhere in the API?  We do allow version mismatch between clients and servers, but there could be client versions that are disallowed.

Reply:
IMO, that's a documentation issue (maybe even just Lustre documentation), not a swagger issue.